tests/validation/validate_kinetic.hpp

```diff
@@ -28,7 +28,7 @@ void run_kinetic_dt(
     };
 
     meta["sim"] = "nestmc";
-    meta["backend"] = backend==backend_policy::use_multicore? "cpu" : "gpu"; // FIXME: fragile
+    meta["backend"] = to_string(backend);
```

Comment:
This is the only remaining thing: for consistency, this key should also be "backend_policy".